report.md

```diff
@@ -867,7 +869,85 @@ This is a best-practice to protect against reentrancy in other modifiers.
 
 
 
-## NC-7: Large literal values multiples of 10000 can be replaced with scientific notation
+## NC-7: Empty Block
+
+Consider removing empty blocks.
+
+- Found in src/AdminContract.sol [Line: 14](tests/contract-playground/src/AdminContract.sol#L14)
+
+	```solidity
+	    function someOtherImportantThing() external nonReentrant onlyOwner {
+	```
+
+- Found in src/EmptyBlocks.sol [Line: 6](tests/contract-playground/src/EmptyBlocks.sol#L6)
```

Comment:
Constructor, receive and fallback are all caught unintentionally here @TilakMaddy . Can you please fix?

Reply:
Thanks so much for reviewing and pointing out ! I have an idea : ) Let me fix it !